rules/windows/privilege_escalation_printspooler_registry_copyfiles.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2020/11/26"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects attempts to exploit privilege escalation vulnerability related to the Print Spooler service via chaining
+multiples primitives to load an arbitrary DLL on the Spoolsv.exe process running as System. For more information refer
+to the following CVE-2020-1030 and verify that the impacted system is patched.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious PrintSpooler Point and Print DLL"
+references = [
+    "https://www.accenture.com/us-en/blogs/cyber-defense/discovering-exploiting-shutting-down-dangerous-windows-print-spooler-vulnerability",
+    "https://github.com/sbousseaden/EVTX-ATTACK-SAMPLES/blob/master/Privilege%20Escalation/privesc_sysmon_cve_20201030_spooler.evtx",
+]
+risk_score = 74
+rule_id = "bd7eefee-f671-494e-98df-f01daf9e5f17"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan = 30s
+[registry where
+ wildcard(registry.path, "HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Print\\Printers\\*\\SpoolDirectory") and
+ wildcard(registry.data.strings, "C:\\Windows\\System32\\spool\\drivers\\x64\\4")]
+[registry where
+ wildcard(registry.path, "HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Print\\Printers\\*\\CopyFiles\\Payload\\Module") and
+ wildcard(registry.data.strings, "C:\\Windows\\System32\\spool\\drivers\\x64\\4\\*")]
```

Comment:
Are we still using the `wildcard` function or do we want to write it like below (haven't tested against real data, but validated the query syntax works)
```
sequence by host.id with maxspan = 30s
[registry where registry.path : "HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Print\\Printers\\*\\SpoolDirectory") and
 registry.data.strings : "C:\\Windows\\System32\\spool\\drivers\\x64\\4"]
[registry where registry.path : "HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Print\\Printers\\*\\CopyFiles\\Payload\\Module" and
 registry.data.strings : "C:\\Windows\\System32\\spool\\drivers\\x64\\4\\*")]
```

Reply:
tested only on 7.10 with wildcard(), not sure if this new syntax will work or not. 

![image](https://user-images.githubusercontent.com/64742097/100669846-4b684880-335e-11eb-94b7-cd3a6907f6c8.png)